get-involved/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+hero-image: url(../assets/images/stock-images/pie-staff.jpg)
+hero-message: Get Involved!
+---
+
+<!-- WHAT WE DO -->
+<div class="container">
+    <h1 style="text-align: center">Join PiE Staff</h1>
+    <!--<h2>WHAT WE DO</h2>-->
+</div>
+<div class="rc-row vertical-align" >
```

Comment:
Also, for these sections: Engineering/Outreach, Competition, PiE Staff Roles, Become a Member - would it be possible to have space in between each so it's not like one contiguous block?